### PR TITLE
update 4.11.13 bundle hash

### PR DESCRIPTION
**Pull request by anjannath**
we had to regenerate 4.11.13 bundles after https://github.com/crc-org/snc/pull/605

### CODE REVIEW

File: pkg/crc/machine/bundle/constants.go [hunk 1]
```diff
@@ -9,22 +9,22 @@ var bundleLocations = map[string]bundlesDownloadInfo{
 	"amd64": {
 		"darwin": {
 			preset.OpenShift: download.NewRemoteFile("https://mirror.openshift.com/pub/openshift-v4/clients/crc/bundles/openshift/4.11.13/crc_vfkit_4.11.13_amd64.crcbundle",
-				"5e86e132e9a2d441dbbfa696c4f3256ade2018de66391d932f560bd5afa1415a"),
+				"bec15ed8f86bd8951940f4e23cbe918594c53accd2dd67fe45b66576bd8de98f"),
```
Comment (praveenkumar) [inline pkg/crc/machine/bundle/constants.go:12]: 👍🏼

**Review by praveenkumar — APPROVED** [review]

**Comment by anjannath** [comment]
/hold

vfkit `arm` bundle has expired certs, `crc start` fails with the following:

```
DEBU Running SSH command: date --date="$(sudo openssl x509 -in /var/lib/kubelet/pki/kubelet-server-current.pem -noout -enddate | cut -d= -f 2)" --iso-8601=seconds
DEBU SSH command results: err: <nil>, output: 2022-11-21T08:33:58+00:00
DEBU Certs have expired, they were valid till: 21 Nov 22 08:33 +0000
DEBU error: Temporary error: certificate /var/lib/kubelet/pki/kubelet-server-current.pem still expired - sleeping 5s
DEBU retry loop: attempt 1
DEBU Running SSH command: date --date="$(sudo openssl x509 -in /var/lib/kubelet/pki/kubelet-server-current.pem -noout -enddate | cut -d= -f 2)" --iso-8601=seconds
DEBU SSH command results: err: <nil>, output: 2022-11-21T08:33:58+00:00
DEBU Certs have expired, they were valid till: 21 Nov 22 08:33 +0000
DEBU error: Temporary error: certificate /var/lib/kubelet/pki/kubelet-server-current.pem still expired - sleeping 5s
```

**Comment by cfergeau** [comment]
> vfkit arm bundle has expired certs, crc start fails with the following:

cert renewal is not working anymore? We need to release a bundle with no expired certs, but good to know if there's also a bug with cert renewal.

**Comment by gbraad** [comment]
What is the impact to continue the release with the current bundle?

**Comment by anjannath** [comment]
> cert renewal is not working anymore? We need to release a bundle with no expired certs, but good to know if there's also a bug with cert renewal.

Its not exactly clear, for the macOS arm bundle which has already expired certs, renewal is failing.. but for the linux amd64 bundle, in the CI the cert rotation test is passing.. [CI logs](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/crc-org_crc/3426/pull-ci-crc-org-crc-main-e2e-crc/1595021621943341056)

**Comment by anjannath** [comment]
> What is the impact to continue the release with the current bundle?

I'd say its blocker since cert rotation (at least on my M1) is failing, users will not be able to start crc

**Comment by anjannath** [comment]
tried to trigger the cert rotation manually by setting system time to 1 year in the future, but it failed to renew the certs, logs are huge so put them in a [gist](https://gist.github.com/anjannath/0d70658bd02cacdf66d90e090757240f)

**Pull request merged**